src/Host/API/Test/Microsoft.R.Host.Client.API.Test.csproj

```diff
@@ -48,10 +48,6 @@
       <Project>{5EF2AD64-D6FE-446B-B350-8C7F0DF0834D}</Project>
       <Name>Microsoft.UnitTests.Core</Name>
     </ProjectReference>
-    <ProjectReference Include="..\..\API\Impl\Microsoft.R.Host.Client.API.csproj">
-      <Project>{1a02711f-3b13-48be-8d45-073386354174}</Project>
-      <Name>Microsoft.R.Host.Client.API</Name>
```

Comment:
ok

---
In reply to: [106054311](https://github.com/Microsoft/RTVS/pull/3302#discussion_r106054311) [](ancestors = 106054311)